steam_wrapper/steam_wrapper.py

```diff
@@ -15,7 +15,7 @@
 
 
 FLATPAK_ID = os.getenv("FLATPAK_ID", "com.valvesoftware.Steam")
-STEAM_PATH = "/app/bin/steam"
+STEAM_PATH = "/app/lib/steam/bin_steam.sh"
```

Comment:
If you follow my suggestion above, this should point at where we moved the original `/app/bin/steam`.